pkg/pillar/hypervisor/containerd.go

```diff
@@ -105,27 +105,27 @@ func (ctx ctrdContext) Delete(domainName string, domainID int) error {
 	return err
 }
 
-func (ctx ctrdContext) Info(domainName string, domainID int) (int, DomState, error) {
+func (ctx ctrdContext) Info(domainName string, domainID int) (int, types.SwState, error) {
 	effectiveDomainID, status, err := containerd.CtrContainerInfo(domainName)
 	if err != nil {
-		return 0, Unknown, logError("containerd looking up domain %s with PID %d resulted in %v", domainName, domainID, err)
+		return 0, types.UNKNOWN, logError("containerd looking up domain %s with PID %d resulted in %v", domainName, domainID, err)
 	}
 
 	if effectiveDomainID != domainID {
 		log.Warnf("containerd domain %s with PID %d (different from expected %d) is %s",
 			domainName, effectiveDomainID, domainID, status)
 	}
 
-	stateMap := map[string]DomState{
-		"running": Running,
-		"created": Blocked,
-		"paused":  Paused,
-		"stopped": Exiting,
-		"pausing": Running,
+	stateMap := map[string]types.SwState{
+		"running": types.RUNNING,
+		"created": types.INSTALLED,
+		"paused":  types.HALTED,
+		"stopped": types.HALTED,
+		"pausing": types.HALTING,
```

Comment:
Do we want more states to represent these differences?

Also the last one seems to change in a semantic way.

Reply:
Good question @eriknordmark -- I am of two minds on this. On one had I'd love to see a more fine grained split, on the other we have quite a few states as it is. What would be your guidance here?